src/js/mvc/view.js

```diff
@@ -27,6 +27,8 @@ export class AbstractView {
 
     }
 
+    onClickSearchBarOverlay() {}
+
```

Comment:
it is not abstract method that could be inherited and extended. Should be deleted from here